api/bnd.bnd

```diff
@@ -2,7 +2,6 @@
     org.eclipse.microprofile.*
 
 Import-Package: \
-    org.eclipse.microprofile.reactive.streams;version="[1.0.0,2)", \
```

Comment:
I believe we still need to specify the right imports coming from reactive streams ops:

```
org.eclipse.microprofile.reactive.streams.operators;version="[1.0.0,2)", \
org.eclipse.microprofile.reactive.streams.operators.spi;version="[1.0.0,2)", \
*
```

Reply:
Those imports are automatically added by the `bnd` plugin. This is the output of `$ bnd print api/target/microprofile-reactive-messaging-api-1.1-SNAPSHOT.jar` after application of this patch:

```
[IMPEXP]
Import-Package
  javax.enterprise.util                  {version=[2.0,3)}
  javax.inject                           
  org.eclipse.microprofile.config        {version=[1.0,2)}
  org.eclipse.microprofile.reactive.messaging {version=[1.0,2)}
  org.eclipse.microprofile.reactive.streams.operators {version=[1.0,2)}
Export-Package
  org.eclipse.microprofile.reactive.messaging {version=1.0, imported-as=[1.0,2)}
  org.eclipse.microprofile.reactive.messaging.spi {version=1.0}
```

Compare that to the current OSGi metadata in the 1.0 release:

```
[IMPEXP]
Import-Package
  javax.enterprise.util                  {version=[2.0,3)}
  javax.inject                           
  org.eclipse.microprofile.config        {version=[1.0,2)}
  org.eclipse.microprofile.reactive.messaging {version=[1.0,2)}
  org.eclipse.microprofile.reactive.streams {version=[1.0.0,2)}
  org.eclipse.microprofile.reactive.streams.operators {version=[1.0,2)}
Export-Package
  org.eclipse.microprofile.reactive.messaging {version=1.0, imported-as=[1.0,2)}
  org.eclipse.microprofile.reactive.messaging.spi {version=1.0}
```

The only difference is that (non-existent) `org.eclipse.microprofile.reactive.streams` package is no longer imported.

Reply:
Sounds good!